frontend/src/routes/team/+page.svelte

```diff
@@ -0,0 +1,43 @@
+<script>
+	let teamMembers = [
```

Comment:
Create a file named `constants.ts` in the team directory right alongside `+page.svelte`. Put the team data in there. Also, you could upload any custom images to a temp storage like https://ibb.co and get the image links from there to insert in the data.